apps/modernization-ui/src/apps/deduplication/patient-merge/landing/matches-requiring-review-table/MatchesRequiringReviewTable.tsx

```diff
@@ -3,27 +3,21 @@ import { useMatchesRequiringReview } from 'apps/deduplication/api/useMatchesRequ
 import { SearchResultPageSizeSelect } from 'apps/search/layout/result/pagination/page-size-select';
 import { SearchResultsShowing } from 'apps/search/layout/result/pagination/showing';
 import { Loading } from 'components/Spinner';
-import { Shown } from 'conditional-render';
 import { format, parseISO } from 'date-fns';
 import { Button } from 'design-system/button';
 import { Pagination } from 'design-system/pagination';
 import { Column, DataTable } from 'design-system/table';
-import { PaginationProvider, Status, usePagination } from 'pagination';
+import { Status, usePagination } from 'pagination';
 import { useEffect, useState } from 'react';
+import { Shown } from 'conditional-render';
+import { Direction, useSorting } from 'sorting';
 import { useNavigate } from 'react-router';
-import { Direction, SortingProvider, useSorting } from 'sorting';
 import styles from './matches-requiring-review.module.scss';
 
 const DATE_FORMAT = 'MM/dd/yyyy h:mm a';
 
 export const MatchesRequiringReviewTable = () => {
-    return (
-        <PaginationProvider pageSize={20}>
-            <SortingProvider>
-                <SortableMatchesRequiringReviewTable />
-            </SortingProvider>
-        </PaginationProvider>
-    );
+    return <SortableMatchesRequiringReviewTable />;
```

Comment:
Issue: since the `PaginationProvider` and `SortingProvider` are now defined in the parent component, there is no need to have this wrapper component. 

You can delete the existing wrapper, and rename the `SortableMatchesRequiringReviewTable` to `MatchesRequiringReviewTable` and export it